front-end/src/Profile.js

```diff
@@ -126,7 +126,9 @@ const ProfileContents = ({ data, updateUserData }) => {
 
   return (
     <>
-      <div id="container">
+      <div className="showContent">
+        <fieldset className="main">
+          <div className="showDetails">
```

Comment:
A few things here: 
I thought that fieldsets were to be used with forms? It doesn't make sense to me to use one here, as opposed to a `div` element

Also, I don't think these class names make much sense to me, as they're wrapping around the profile. Perhaps you could rename them to make them more clear?